modules/sfp_securitytxt.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+# -------------------------------------------------------------------------------
+# Name:        sfp_securitytxt
+# Purpose:     Find and parse .well-known/security.txt files.
+#
+# Author:      Steve Micallef <steve@binarypool.com>
```

Comment:
This should reflect you as the author :)

Reply:
I originally had my name there, but I wasn't sure how to handle the copyright. Should I just replace the name and leave you as the copyright holder?

Reply:
Your name there as the author is fine.